multigeiger/display.cpp

```diff
@@ -64,18 +61,18 @@ void setup_display(bool loraHardware) {
   display_start_screen();
 }
 
-void clearDisplayLine(int line) {
+void clear_displayline(int line) {
```

Comment:
scope creep again.

a better way to deal with such stuff is to resist scope creep in a PR and just file a ticket for a style refactoring.
and then do that in a separate PR solving it globally.

Reply:
Guilty as charged. I just wanted to change the local functions in display.cpp but got indeed carried away. 
Sorry, will avoid this next time!